import-export-cli/integration/testutils/policy_testUtils.go

```diff
@@ -0,0 +1,389 @@
+	t.Helper()
+
+	// Setup apictl envs
+	base.SetupEnv(t, args.SrcAPIM.GetEnvName(), args.SrcAPIM.GetApimURL(), args.SrcAPIM.GetTokenURL())
+	base.SetupEnv(t, args.DestAPIM.GetEnvName(), args.DestAPIM.GetApimURL(), args.DestAPIM.GetTokenURL())
+
+	// Export policy from env 1
+	base.Login(t, args.SrcAPIM.GetEnvName(), args.CtlUser.Username, args.CtlUser.Password)
+	policyName := fmt.Sprintf("%v", args.Policy[policyNameKey])
+
+	exportedOutput, err := exportThrottlePolicy(t, policyName, args)
+	assert.Nil(t, err, "Error while exporting the Throttling Policy")
+
+	args.ImportFilePath = base.GetExportedPathFromOutput(exportedOutput)
+	assert.True(t, base.IsFileAvailable(t, args.ImportFilePath))
+	args.SrcAPIM.DeleteThrottlePolicy(fmt.Sprintf("%v", args.Policy[policyIDKey]), policyType)
+
+	// Import Throttling Policy to env 2
+	base.Login(t, args.DestAPIM.GetEnvName(), args.CtlUser.Username, args.CtlUser.Password)
+	importedOutput, err := importThrottlePolicy(t, args)
+	assert.Nil(t, err, "Error while importing the Throttling Policy")
+	assert.Contains(t, importedOutput, "Successfully imported")
+	// Give time for newly imported Throttling Policy to get indexed
+	base.WaitForIndexing()
+
+	// Get Throttle Policy from env 2
+	importedPolicy, _ := getThrottlingPolicyByName(t, args, adminUsername, adminPassword, policyName, policyType)
+	// Validate env 1 and env 2 policy is equal
+	validatePoliciesEquality(t, true, args, importedPolicy)
+}
+
+// ValidateThrottlePolicyImportUpdate : Validates importing existing Throttling Policy
+func ValidateThrottlePolicyImportUpdate(t *testing.T, args *PolicyImportExportTestArgs, adminUsername, adminPassword, policyType string) {
+	t.Helper()
+
+	// Setup apictl envs
+	base.SetupEnv(t, args.SrcAPIM.GetEnvName(), args.SrcAPIM.GetApimURL(), args.SrcAPIM.GetTokenURL())
+	base.SetupEnv(t, args.DestAPIM.GetEnvName(), args.DestAPIM.GetApimURL(), args.DestAPIM.GetTokenURL())
+
+	// Export policy from env 1
+	base.Login(t, args.SrcAPIM.GetEnvName(), args.CtlUser.Username, args.CtlUser.Password)
+	policyName := fmt.Sprintf("%v", args.Policy[policyNameKey])
+
+	exportedOutput, _ := exportThrottlePolicy(t, policyName, args)
+	args.ImportFilePath = base.GetExportedPathFromOutput(exportedOutput)
+	assert.True(t, base.IsFileAvailable(t, args.ImportFilePath))
+	args.SrcAPIM.DeleteThrottlePolicy(fmt.Sprintf("%v", args.Policy[policyIDKey]), policyType)
+
+	// Import Throttling Policy to env 2
+	base.Login(t, args.DestAPIM.GetEnvName(), args.CtlUser.Username, args.CtlUser.Password)
+	importedOutput, err := importThrottlePolicy(t, args)
+	assert.Nil(t, err, "Error while importing the Throttling Policy")
+	assert.Contains(t, importedOutput, "Successfully imported")
+	// Give time for newly imported Throttling Policy to get indexed
+	base.WaitForIndexing()
+
+	changeExportedThrottlePolicyFile(t, args)
+	// Import Throttling Policy to env 2 to update
+	output, err := importThrottlePolicy(t, args)
```

Comment:
Here, the update flag should be true. Hence, add a line before this to make the args.Update = true explicitly.

Reply:
Fixed with 53f8cc8609cb4711d71c229466ebbd0f9c8f10e5